Sources/MapboxMaps/Style/Style+Localization.swift

```diff
@@ -0,0 +1,88 @@
+@_implementationOnly import MapboxCommon_Private
+
+/// Support for localization of labels
+extension Style {
+
+    /// This function creates an expression that will localize the `textField` property of a `SymbolLayer`
+    /// - Parameter locale: A `SupportedLanguage` based `Locale`
+    internal func localizeLabels(into locale: Locale) {
```

Comment:
Nav SDK has a requirement that not all labels are localized I think.. They want to exempt road labels for example from any localization.

To help them do this, it's probably best to break this function up into some functions like this:

```swift
public func localizeAllLabels(into locale: Locale) { .. }

public func localizeLabel(withId id: String, into locale: Locale) { .. } 

```

Reply:
@julianrex @tobrun This feels like something we need to align on. This functionality does not exist on Android